src/components/shared/TreeView.js

```diff
@@ -190,10 +190,8 @@ class TreeViewRowScrolledColumns<
    * `target` instead of `currentTarget`.
    */
   _onClick = (event: { target: Element } & SyntheticMouseEvent<Element>) => {
-    const { nodeId, isExpanded, onToggle, onClick } = this.props;
-    if (event.target.classList.contains('treeRowToggleButton')) {
-      onToggle(nodeId, !isExpanded, event.altKey === true);
-    } else {
+    const { nodeId, onClick } = this.props;
+    if (!event.target.classList.contains('treeRowToggleButton')) {
```

Comment:
This change is fine, but i'd like this method `onClick` to be renamed to `onMouseDown` because this is the event that triggers this. Of course you'll need to change it in the `onMouseDown` prop too !